src/modules/AdminModule.cpp

```diff
@@ -221,8 +224,6 @@ bool AdminModule::handleReceivedProtobuf(const meshtastic_MeshPacket &mp, meshta
             nodeDB->setLocalPosition(r->set_fixed_position);
             config.position.fixed_position = true;
             saveChanges(SEGMENT_DEVICESTATE | SEGMENT_CONFIG, false);
-            // Send our new fixed position to the mesh for good measure
```

Comment:
This seems accidental removal?

Reply:
That's what I get for copy and pasting changes from another branch downstream of master changes